<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CovidMapManagerSystem</class>
 <widget class="QDialog" name="CovidMapManagerSystem">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>疫情地图管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GUI.qrc">
    <normaloff>:/GUI/icon.png</normaloff>:/GUI/icon.png</iconset>
  </property>
  <widget class="QFrame" name="CovidMapBackgroung">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(48, 50, 190, 93), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="UI_login_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>432</width>
      <height>768</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/GUI/UI_login_bg.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="UI_login_hint_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>432</width>
      <height>768</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/GUI/UI_login_hint_1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AdminButton">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>91</y>
      <width>115</width>
      <height>183</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/GUI/UI_login_btn_admin_1.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="GuestButton">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>91</y>
      <width>115</width>
      <height>183</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/GUI/UI_login_btn_guest_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="UserButton">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>91</y>
      <width>115</width>
      <height>183</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/GUI/UI_login_btn_user_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="QuitButton">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>8</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/GUI/UI_login_btn_quit_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections/>
</ui>
